packages/manager/modules/sms/src/sms/batches/create/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+import '@ovh-ux/ui-kit';
+import '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@uirouter/angularjs';
+
+import component from './telecom-sms-batches-create.component';
+import routing from './routing';
+
+const moduleName = 'ovhManagerSmsBatchesCreate';
+
+angular
+  .module(moduleName, ['ngOvhTelecomUniverseComponents', 'oui', 'ui.router'])
```

Comment:
No `pascalprecht.translate` ?

Reply:
Hmm it seems that the whole `sms` module hasn't `angular-translate` (nor its own dependencies) declared as peerDependency. + no injection of the dep in all the sms angularjs modules.

I'm ok with your request, but maybe this should be tackle down in another PR, to update the `sms` app. What do you think about ?